docs/release_notes/sat_2.6_release_notes.md

```diff
@@ -0,0 +1,73 @@
+# Changes in SAT 2.6
+
+The 2.6.4 version of the SAT product includes:
+
+- Version 3.24.0 of the `sat` python package and CLI.
+- Version 2.0.0-1 of the `sat-podman` wrapper script.
+- Version 1.6.1 of the `sat-install-utility` container image.
+
+## New `sat` commands
+
+No new `sat` commands were added in SAT 2.6.
+
+## Changes to `sat bootsys`
+
+- Functionality has been added to the `platform-services` and `cabinet-power`
+  stages of `sat bootsys boot` which allows SAT to automatically re-create
+  Kubernetes CronJobs which may have become stuck during shutdown, boot, or
+  reboot.
+
+- `sat bootsys boot` will more reliably determine if the `hms-discovery` cronjob
+  has been scheduled during the `cabinet-power` stage. This same functionality
+  is also used by `sat swap blade`.
+
+- SAT will use the BatchV1 Kubernetes API to manipulate CronJobs, instead of the
+  BatchV1Beta1 API as previously.
+
+- `sat bootsys` will now log the ID of all BOS sessions when performing BOS
+  operations. A warning will be logged for any BOS sessions with failed
+  components.
+
+## Changes to `sat bootprep`
+
+- Wildcard matching was added for images in bootprep input files. This can be
+  used similarly to how prefix filters have been used in previous versions of
+  SAT. For more information, see
+  [Define IMS Images](../usage/sat_bootprep.md#define-ims-images).
+
+## Changes to other `sat` commands
+
+- A new `s3.cert_verify` option can be added to the SAT configuration file which
+  controls whether certificate verification is performed when accessing S3.
+
+- Log messages spanning multiple lines will print the log level on each line,
+  instead of only at the beginning of the message.
+
+- When certificate verification is disabled for CSM API requests, only a single
+  warning will be printed at the beginning of SAT's invocation instead of for
+  each request.
+
+## CRUS Removal
+
+Support for the Compute Rolling Upgrade Service (CRUS) has been removed. This
+change affects the following SAT commands which interacted with CRUS:
+
+- `sat bootsys`
+
+Affected commands will no longer interact with CRUS.
+
+## Multi-tenancy
+
+- SAT 2.6 supports supplying tenant information to CSM services in order to
+  allow tenant admins to use SAT within their tenant. For more information, see
+  [Configure multi-tenancy](../usage/multi-tenancy.md).
+
+## Security
```

Comment:
We also addressed CVE-2023-32681 in requests [in cray-sat:3.24.0](https://github.com/Cray-HPE/sat/blob/release/3.24/CHANGELOG.md#security).